Competition2017/src/competition/subsystems/shooter_belt/commands/BaseShooterBeltCommand.java

```diff
@@ -1,14 +1,23 @@
 package competition.subsystems.shooter_belt.commands;
 
 import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
 import xbot.common.command.BaseCommand;
 
 public abstract class BaseShooterBeltCommand extends BaseCommand {
     
     protected final ShooterBeltSubsystem shooterBeltSubsystem;
+    protected final ShooterWheelSubsystem shooterWheelSubsystem;
```

Comment:
What do you think about making a `BaseChooterBeltAndWheelCommand` which has this in it instead of a nullable field here?